distribution/changelog.txt

```diff
@@ -3,6 +3,7 @@
 - Fix: [#19296] Crash due to a race condition for parallel object loading.
 - Fix: [#19756] Crash with title sequences containing no commands.
 - Fix: [#19767] No message when path is not connected to ride exit and is therefore unreachable for mechanics.
+- Fix: [#19231] Crash due to null pointer to previously deleted banner in tile copy/paste functionality
```

Comment:
This should be moved up all the way to the top, so that the issue/pr numbers are incremental.